### PR TITLE
Update venmo.create method options

### PR DESCRIPTION
**Pull request by Srinathdhamodharn**
Update venmo.create method options in venmo.d.ts for types/braintree-web

JS Code Snippet:

`braintree.venmo.create({ client: clientInstance, allowDesktop: true, mobileWebFallBack: true, allowDesktopWebLogin: true, paymentMethodUsage: 'multi_use' }, function (venmoErr, venmoInstance) { if (venmoErr) { console.error('Error creating Venmo:', venmoErr); return; } }); `

Documentation Link - https://developer.paypal.com/braintree/docs/guides/venmo/client-side/javascript/v3/

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## braintree-web ([<kbd>unpkg</kbd>](https://unpkg.com/browse/braintree-web@latest/))
was missing the following properties:
1. masterpass
1. paymentRequest
1. visaCheckout
1. sepa
1. preferredPaymentMethods
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1f2dbaa1f52b7e5f5f600a4c5f765d65c893924c
</p>

File: types/braintree-web/venmo.d.ts [hunk 1]
```diff
@@ -98,6 +98,10 @@ export function create(options: {
     ignoreHistoryChanges?: boolean | undefined;
     profileId?: string | undefined;
     deepLinkReturnUrl?: string | undefined;
+    allowDesktopWebLogin?: boolean | undefined;
+    allowDesktop?: boolean | undefined;
+    mobileWebFallBack?: boolean | undefined;
+    paymentMethodUsage?: string | undefined;
```
Comment (daelmaak) [inline types/braintree-web/venmo.d.ts:104]: Please make the type of `paymentMethodUsage` stricter, see the possible options in https://braintree.github.io/braintree-web/current/module-braintree-web_venmo.html#.create

File: types/braintree-web/venmo.d.ts [hunk 2]
```diff
@@ -98,6 +98,10 @@ export function create(options: {
     ignoreHistoryChanges?: boolean | undefined;
     profileId?: string | undefined;
     deepLinkReturnUrl?: string | undefined;
+    allowDesktopWebLogin?: boolean | undefined;
+    allowDesktop?: boolean | undefined;
+    mobileWebFallBack?: boolean | undefined;
```
Comment (daelmaak) [inline types/braintree-web/venmo.d.ts:103]: The `| undefined` is redundant when the property is declared with `?`.

**Review by daelmaak — APPROVED** [review]
Thank you, few things to consider, mainly the stricter typing.

File: types/braintree-web/venmo.d.ts (hunk 1 above)
Comment (Srinathdhamodharn) [reply types/braintree-web/venmo.d.ts]: Update the types paymentMethodUsage Property.

File: types/braintree-web/venmo.d.ts (hunk 2 above)
Comment (Srinathdhamodharn) [reply types/braintree-web/venmo.d.ts]: Removed the redundant undefineds.

**Comment by Srinathdhamodharn** [comment]
@typescript-bot  Could you please rerun the CI / test .

**Comment by Srinathdhamodharn** [comment]
@daelmaak I have resolved the PR Comments, could you please re-review and approve it.

**Review by daelmaak — APPROVED** [review]
Thanks for the changes

**Comment by daelmaak** [comment]
Btw CI is failing probably due to GH's degraded performance.

**Comment by Srinathdhamodharn** [comment]
> Btw CI is failing probably due to GH's degraded performance.

@daelmaak CI passed now, I have added some description for allowed values for Payment Method Usage

**Review by daelmaak — APPROVED** [review]
You can merge it

**Pull request merged**